src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.wasm.cs

```diff
@@ -320,8 +337,12 @@ private void TryProcessScrollTo()
 			}
 		}
 
-		private void OnScroll(object sender, EventArgs args)
+		private bool OnScroll(object sender, RoutedEventArgs routedEventArgs)
 		{
+			if (Scroller?.CancelNextNativeScroll ?? false)
+			{
+				return true;
+			}
```

Comment:
I've the feeling this is too late: this event is raised by the platform when it has scrolled, so we can update the managed properties and raise the event accordingly ... ignoring it here would just drive to be out-of-sync with the native scroll offsets?

Reply:
MDN seems to agree with you, but I've tested this on WASM and it works (I've tested on Chromium). I can't find the link right now, but if I remember correctly, someone online also suggested this, so I guess we're fine?

Reply:
Ok, we need to add some details in comments then. I would also validate this at least on Firefox which is not chromium based.

Reply:
Just tested firefox, also works. Not sure what to add to the comments, we just `preventDefault` the event. If you have something more eloquent, feel free to add a comment there.